packages/es-components/src/components/containers/loading-skeleton/LoadingSkeleton.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled, { ThemeProvider } from 'styled-components';
+import tinycolor from 'tinycolor2';
+import { useTheme } from '../../util/useTheme';
+
+const SkeletonContainer = styled.div`
+  position: relative;
+  background-color: ${({ theme: { skeleton } }) => skeleton.shimmerColor};
+
+  &::after {
+    animation: loading-keyframes 1.5s linear infinite;
```

Comment:
This seems a bit fast for my eyes. I tried it with 2s and it looks good with the benefit of not giving me a dizzy feeling. :)

Reply:
Yes it's a subjective view but if you could run that past Keoni I'd appreciate it.